libkbfs/interfaces.go

```diff
@@ -349,6 +349,10 @@ type KeybaseService interface {
 	// instance. No other methods may be called after this is
 	// called.
 	Shutdown()
+
+	// EstablishMountDir asks the service for the current mount path
+	// and sets it if not establishe
+	EstablishMountDir(ctx context.Context) (string, error)
```

Comment:
Also, we like to make `Shutdown` the last thing in an interface, so please move this above it.